src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -399,11 +399,11 @@
     <Reference Include="System.Diagnostics.Debug" />
     <Reference Include="System.Diagnostics.Tracing" />
     <Reference Include="System.IO.FileSystem" />
+    <Reference Include="System.Memory" />
     <Reference Include="System.Net.NameResolution" />
     <Reference Include="System.Net.Primitives" />
     <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Runtime" />
-    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
```

Comment:
Are you going to delete this from all netcoreapp projects?

```
  src\System.Net.Primitives\src\System.Net.Primitives.csproj(200):    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
  src\System.Net.Security\src\System.Net.Security.csproj(413):    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
  src\System.Net.WebSockets\src\System.Net.WebSockets.csproj(39):    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
  src\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj(146):    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
```

Reply:
I didn't find any use of CompilerServices.Unsafe APIs in those projects (which is what I was searching for). I am not sure why they referenced it. I will remove them.